pkg/identityserver/bunstore/invitation_store.go

```diff
@@ -195,58 +194,31 @@ func (s *invitationStore) GetInvitation(ctx context.Context, token string) (*ttn
 		}
 		return nil, err
 	}
-
-	pb, err := invitationToPB(model)
-	if err != nil {
-		return nil, err
-	}
-
-	return pb, nil
+	return invitationToPB(model)
 }
 
 func (s *invitationStore) SetInvitationAcceptedBy(
-	ctx context.Context, token string, accepter *ttnpb.UserIdentifiers,
+	ctx context.Context, validation *ttnpb.Invitation, usrIDs *ttnpb.UserIdentifiers,
```

Comment:
I don't fully understand what we gain by passing the full `*ttnpb.Invitation` here - we use only the token.